### PR TITLE
Clarify best practices for operation names

### PR DESCRIPTION
**Pull request by bensigelman**
cc @dominikh

### CODE REVIEW

**Comment by dominikh** [comment]
LGTM

**Comment by bensigelman** [comment]
I'll leave this open for at least 24h in case anyone watching has concerns.

File: spec.md [hunk 1]
```diff
@@ -46,13 +46,17 @@ A **Trace** represents the potentially distributed, potentially concurrent data/
 
 ## Spans
 
-A **Span** represents a logical unit of work in the system that has a start time and a duration. Spans may be nested and ordered to model causal relationships. Each span has an **operation name**, a presumably human-readable string which concisely represents the work done by the span (e.g., an RPC method name, a function name, or the name of a subtask or stage within a larger computation).
+A **Span** represents a logical unit of work in the system that has a start time and a duration. Spans may be nested and ordered to model causal relationships.
+
+### Operation Names
+
+Each Span has an **operation name**, a human-readable string which concisely represents the work done by the Span (e.g., an RPC method name, a function name, or the name of a subtask or stage within a larger computation). The operation name should be **the most general (i.e., least specific) string that identifies a (statistically) interesting class of Span instances**; more specific sub-classes should be described using [Tags](#tags).
```
Comment (yurishkuro) [inline spec.md]: Maybe give example?
  * "get" - too general
  * "get_account/12345" - too specific
  * "get_account" - good

**Comment by yurishkuro** [comment]
LGTM

File: spec.md (hunk 1 above)
Comment (bensigelman) [inline spec.md]: I did something similar, good suggestion.

**Comment by bensigelman** [comment]
Okay, merging. Thanks!

**Pull request merged**